src/content/structured/get-started/a-design-system.mdx

```diff
@@ -12,7 +12,7 @@ contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/struc
 
 ## Our reasons behind building a Design System
 
-We love what we've built. It’s already helped MI6, MI5, and GCHQ to create powerful and accessible capabilities that keep the UK and its partners safe and prosperous.
```

Comment:
We've lost our love for the project ;( 
(this is fine, feels more professional now)